knowledge-base/actions/pragatheeswarans/yarn-audit-action/action-security.yml

```diff
@@ -0,0 +1,11 @@
+name: "Yarn Audit"
+github-token:
+  action-input:
+    input: GITHUB_TOKEN
+    is-default: false
+  permissions:
```

Comment:
I think there is no `actions: read` permissions, please remove it.
and also while specifying reasons be more specific like for `issues-reason`  update it to, `to create issue with security vulnerabilities`